docs/content/protocol/core-contracts/flow-id-table-staking.mdx

```diff
@@ -46,20 +46,34 @@ Documentation for delegating with unlocked tokens will be updated when unlocked
 # Events
 
 The `FlowIDTableStaking` contract emits an event whenever an important action occurs.
+See the [staking events Documentation](../../staking/events) for more information about each event.
```

Comment:
I tested this and the link took me to `protocol/core-contracts/flow-id-table-staking/events` when it should have taken me to `staking/events`. How do we link it correctly?